ProyectoFinal/src/logico/BolsaTrabajo.java

```diff
@@ -135,6 +142,22 @@ public void contratarPersonal(Personal persona, SolicitudEmpresa solicitudEmpres
 					if(idSolicitudPersonal.equalsIgnoreCase(solicitud.getId())){
 						solicitud.setEstado(EstadoSolicitudPersonal.SATISFECHA);
 						persona.setIdSolicitudPersonalContratacion(idSolicitudPersonal);
+						if(persona.getGenero().equalsIgnoreCase("Femenino")) {
+							cantPersonalFem++;
+						}
+						else if(persona.getGenero().equalsIgnoreCase("Masculino")) {
+							cantPersonalMasc++;
+						}
+						
+						if(persona.toString().equalsIgnoreCase("Universitario")) {
```

Comment:
```suggestion
						if(solicitud.getTipoPersonal().equalsIgnoreCase("Universitario")) {
```
La solicitud es que contiene los datos del tipo de personal.